pubtools/_quay/push_docker.py

```diff
@@ -178,6 +178,8 @@ def get_operator_push_items(self):
             if not item.metadata.get("com.redhat.openshift.versions"):
                 msg = "Push item {0} doesn't specify 'com.redhat.openshift.versions'".format(item)
                 raise BadPushItem(msg)
+            if not item.claims_signing_key and docker_push_items:
+                item.claims_signing_key = docker_push_items[0].claims_signing_key
```

Comment:
I think it would rather make sense to do this change here: pubd/pubd/push/adapter.py